contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -332,6 +333,38 @@ def _set_capture_options(self, collect_thread_states: bool, collect_system_memor
             logging.info('Toggling "Collect memory usage and page faults information" checkbox')
             collect_system_memory_usage_checkbox.click_input()
 
+        logging.info('Setting sampling period to {}.'.format(sampling_period_ms))
```

Comment:
Nit: I think this method has become way too large and should be broken into smaller methods (e.g. _set_sampling_period, _set_frame_pointer_unwinding and so on).

Reply:
Done.